modules/tinymce/src/core/test/ts/module/test/PasteEventUtils.ts

```diff
@@ -70,7 +70,12 @@ const pWaitForAndAssertInputEvents = async (beforeinputEvent: SingletonEvent<Inp
   const assertInputEvent = (): void =>
     inputEvent.on((e) => {
       assert.equal(e.inputType, 'insertFromPaste', 'beforeinput event type should be "insertFromPaste"');
-      assert.isNull(e.data, 'input event data should be null');
+      // TINY-11373: Chromium >= 129, e.data is no longer null when pasting plain text
+      if (isNative && browser.isChromium() && browser.version.major >= 129) {
+        assert.isNotNull(e.data, 'input event data should not be null');
+      } else {
+        assert.isNull(e.data, 'input event data should be null');
+      }
```

Comment:
Maybe we don't need to check `e.data` at all 🤔  oh well this will be fine

Reply:
Just making sure that we don't forget about this 🤫